homeassistant/components/lock/zwave.py

```diff
@@ -182,11 +185,11 @@ def clear_usercode(service):
             break
 
     if value.command_class != zwave.const.COMMAND_CLASS_DOOR_LOCK:
-        return
+        return None
     if value.type != zwave.const.TYPE_BOOL:
-        return
+        return None
     if value.genre != zwave.const.GENRE_USER:
-        return
+        return None
     if node.has_command_class(zwave.const.COMMAND_CLASS_USER_CODE):
```

Comment:
They are registered conditionally by accessing `node`, so must be done on OZW thread.